scripts/add-node-in-cluster

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
nit: I'd keep all the test scenarios inside tests/, maybe we can have a script folder in there too

Just a spare thought, have you ever played with https://github.com/kward/shunit2 ? maybe we can simplify some of e2e tests with that or either bats

Reply:
> nit: I'd keep all the test scenarios inside tests/, maybe we can have a script folder in there too. This looks like an e2e test, so I'd go ahead creating a specific folder so we can add more scenarios as we go.

I can do this, no problem.

> Just a spare thought, have you ever played with https://github.com/kward/shunit2 ? maybe we can simplify some of e2e tests with that or either bats

Just had a quick look (I already had a look some years ago) but according to the readme it's more to test a shell script "If you have ever had the desire to write a unit test for a shell script, shUnit2 can do the job.". I don't see how it could simplify the test here TBH. The bootstrapping through Rancher needs to be tested, not the shell script itself.

Reply:
> > Just a spare thought, have you ever played with https://github.com/kward/shunit2 ? maybe we can simplify some of e2e tests with that or either bats
> 
> Just had a quick look (I already had a look some years ago) but according to the readme it's more to test a shell script "If you have ever had the desire to write a unit test for a shell script, shUnit2 can do the job.". I don't see how it could simplify the test here TBH. The bootstrapping through Rancher needs to be tested, not the shell script itself.

It can used like bats to organize more complex scenarios, I've been using it successfully with luet, see e.g. https://github.com/mudler/luet/blob/master/tests/integration/01_simple.sh 
but that's outside of the PR scope, it's fine for now to group it somewhere

Reply:
that would be awesome!